### PR TITLE
Add label sync cron job and add back label which has external dependency 

### PR DESCRIPTION
**Pull request by kunmingg**
Add config for label_sync cron job, and add "do-not-merge/work-in-progress" back to fix prow merge.
Details see: https://github.com/kubeflow/testing/issues/102

### CODE REVIEW

File: label_sync/kubeflow_label.yml [hunk 1]
```diff
@@ -102,8 +102,8 @@ labels:
   name: status/icebox
 - color: ffa500
```
Comment (jlewi) [inline label_sync/kubeflow_label.yml]: Should we change the status/approved label to approved and also add label lgtm? Those labels are used by the k8s-ci-bot. So status/approved is duplicative. I think by adding them here we can ensure consistent colors.

**Comment by jlewi** [comment]
Can you also add label: "area/usage" for issues related to reporting and metrics.

**Comment by kunmingg** [comment]
@jlewi config updated and tested.
Sample action on "kubeflow/community":
```
time="2018-04-10T10:38:25-07:00" level=info msg=migrate from="status/approved" repo=community to=approved
time="2018-04-10T10:38:25-07:00" level=info msg=migrate from="status/in progress" repo=community to="do-not-merge/work-in-progress"
time="2018-04-10T10:38:25-07:00" level=info msg=migrate from="status/lgtm" repo=community to=lgtm
time="2018-04-10T10:38:25-07:00" level=info msg=recolor color=2efca4 label=lgtm repo=community
time="2018-04-10T10:38:25-07:00" level=info msg=create label="area/usage" repo=community
time="2018-04-10T10:38:25-07:00" level=info msg=recolor color=03db12 label=approved repo=community
```

**Pull request merged**